src/routes/spotify/playlist.svelte.ts

```diff
@@ -60,6 +60,12 @@ export const Playlist = () => {
     }, []);
 
   const enqueue = async (at: AlbumTrack) => {
+    if (album.src == at.album.src && track.src == at.track.src) {
+      return;
+    }
+    if (queue.find((q) => q.albumSrc == at.album.src && q.trackSrc == at.track.src)) {
```

Comment:
@nzoschke is this all?! haha it's working in tests, but wondering if I'm missing something

Reply:
Seems right... If they queue the current song that is playing, or something in the queue, don't actually queue